Documentation/getting-started-ubuntu-ppa.md

```diff
@@ -30,6 +31,10 @@ $ sudo apt install nnstreamer
 * nnstreamer-tensorflow : Allows to use TensorFlow models in a pipeline. (From tensorflow 1.13.1 by default)
 * nnstreamer-tensorflow-lite : Allows to use TensorFlow-lite models in a pipeline. (From tensorflow 1.13.1 by default)
 
+For a full list of plugins run:
+```
+lz4cat -d /var/lib/apt/lists/ppa.launchpad.net_nnstreamer_ppa_ubuntu_dists_*_Packages.lz4 | grep "Package:"
```

Comment:
In general, ```$ apt-cache search nnstreamer``` will do what you intended to do.
Besides, in my Ubuntu machine, this Packages file is not compressed.

Reply:
I like this suggestion, just wish it also included all of the other packages like `ssat`.